pkg/ruler/manager_test.go

```diff
@@ -40,21 +45,36 @@ func TestSyncRuleGroups(t *testing.T) {
 				User:      user,
 			},
 		},
+		user1: {
+			&rulespb.RuleGroupDesc{
+				Name:      "group1",
+				Namespace: "ns",
+				Interval:  1 * time.Minute,
+				User:      user1,
+			},
+		},
 	}
 	m.SyncRuleGroups(context.Background(), userRules)
 
 	mgr := getManager(m, user)
 	require.NotNil(t, mgr)
-
 	test.Poll(t, 1*time.Second, true, func() interface{} {
 		return mgr.(*mockRulesManager).running.Load()
 	})
 
+	mgr1 := getManager(m, user1)
```

Comment:
[nit] Follow up of my comment above:

```suggestion
	mgr2 := getManager(m, user2)
```